user-guide/Advanced_Modules/User Defined APIs/Triggering_an_API.md

```diff
@@ -0,0 +1,206 @@
+
+To trigger an API, you can send an HTTP or HTTPS request to the UserDefinableApiEndpoint.
+
+### Building the HTTP request
+
+To test sending a request, you can use a tool like Postman.
+
+#### URL
+
+To send a request to UserDefinableApiEndpoint, use the following URL: `http(s)://<HOSTNAME>/api/custom/ROUTE_HERE`.
+Examples:
+
+```
+http://staging-agent-b3/api/custom/switchers/status
+https://production-agent-a1/api/custom/encoder/main/status
+```
+
+#### HTTP Methods
+
+ The following HTTP methods are supported: GET, PUT, POST, DELETE.
+
+ > [!NOTE]
+ > An API is free to choose if it supports one or more of the aforementioned methods.
+
+#### Authentication
+
+To authenticate yourself to the API, you add a `Bearer` Authorization header to your request that contains the [secret](xref:UD_APIs_Objects_ApiToken#secret). See an example of how the header should look like below:
+
+```
+Bearer 24te8bRBOhrDj6TkXkim4qh9GpDXsdGudib57j7C/CU=
+```
+
+#### Content-Type
+
+In case the API needs input from the user using the HTTP body, you need to specify a content-type.
+The endpoint supports the following content-types:
+
+- application/json
+- application/xml
+- text/xml
+- text/plain
+
+> [!NOTE]
+> Note that it is also supported to add no Content-Type header to your request, in this case, it is expected that the body is empty.
+
+#### Content-Length
+
+The Content-Length header should be calculated and filled in automatically depending on how you send the request. It contains the length of your body in bytes. It is important that this is correct, as UserDefinableApiEndpoint will only read the amount of bytes of the body that are specified in the Content-Length header.
+
+> [!IMPORTANT]
+> The endpoint limits request sizes up to 30 MB.
+
+#### Body
+
+An API can expect input. This input is passed in the body. The format of the input is defined in the definition of the API. If the `ApiDefinition` is set to accept parameters, you are expected to pass them as a JSON in a key-value format. More info about that can be found [here](xref:UD_APIs_Define_API#parameters). The body is expected to be encoded in UTF-8.
+
+### The response
+
+The API returns an HTTP status code indicating the status of the request and a body. In case some input from the user is missing, or the user sends a request with a wrong HTTP method. The API can return an HTTP status code indicating the error and a body with more information. The endpoint itself can also return errors with corresponding status code to indicate something went wrong. See below.
```

Comment:
Don't really understand this sentence, something seems missing after 'or the user sends a request with a wrong HTTP method'